app/feedme/approval/schemas.py

```
@@ -0,0 +1,382 @@
+    # AI extraction metadata
+    extraction_confidence: float = Field(..., ge=0.0, le=1.0)
+    ai_model_used: str = Field(..., min_length=1)
+    extraction_method: str = Field(default="ai")
+    
+    # Optional categorization
+    issue_category: Optional[str] = Field(None, max_length=50)
+    tags: Optional[List[str]] = Field(default_factory=list)
+    metadata: Optional[Dict[str, Any]] = Field(default_factory=dict)
+
+    @validator('question_text', 'answer_text')
+    def validate_text_content(cls, v):
```

Pylint (3.3.7)

[error] Method 'validate_text_content' should have "self" as first argument

(E0213)